exporters/common/src/main/resources/META-INF/native-image/io.opentelemetry.opentelemetry-exporter-common/reflect-config.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "name":"io.opentelemetry.sdk.common.export.AutoValue_RetryPolicy",
```

Comment:
maybe add comment here about why we're adding it even though there's no reflective usage?

Reply:
JSON doesn't typically support comments but I can give it a try

Reply:
ugh ya

Reply:
no worries if it doesn't work

Reply:
```
Error: Error parsing reflection configuration in jar:file:/Users/jberg/code/open-telemetry/opentelemetry-java/exporters/common/build/libs/opentelemetry-exporter-common-1.35.0-SNAPSHOT.jar!/META-INF/native-image/io.opentelemetry.opentelemetry-exporter-common/reflect-config.json:
line 3 column 4 Expected , or } but found /
```
No dice